packages/cli/src/transforms/toktx.ts

```diff
@@ -300,17 +309,22 @@ function createParams(
 		params.push('--resize', `${width}x${height}`);
 	}
 
+	// See https://github.com/donmccurdy/glTF-Transform/pull/389/commits/c4935a7348a401587b8d3bf1319bc8481d90ac8e#r733235207
```

Comment:
this linked commit doesn't exist any longer to reference the comment

Reply:
Can ignore, it was fixed in a PR after this. I had just ended up looking at this PR as linked from the release note. !